src/masternodes/rpc_proposals.cpp

```diff
@@ -123,6 +123,24 @@ UniValue proposalVoteToJSON(const CProposalId &propId, uint8_t cycle, const uint
     return ret;
 }
 
+void proposalVoteAccounting(const CProposalVoteType &vote, uint256 propId, std::map<std::string, VotingInfo> &map) {
+    const auto voteString = CProposalVoteToString(vote);
+
+    if (map.find(propId.GetHex()) == map.end())
+        map.emplace(propId.GetHex(), VotingInfo{0, 0, 0, 0, 0, 0});
+
+    auto entry = &map.find(propId.GetHex())->second;
+
+    if (voteString == "YES")
+        ++entry->votesYes;
+    else if (voteString == "NEUTRAL")
+        ++entry->votesNeutral;
+    else if (voteString == "NO")
+        ++entry->votesNo;
+
```

Comment:
Feels like we're missing some invalid votes here.
`getgovproposal` uses 
```
    int targetHeight;
    if (prop->status == CProposalStatusType::Voting) {
        targetHeight = view.GetLastHeight() + 1;
    } else {
        targetHeight = prop->cycleEndHeight;
    }

    std::set<uint256> activeMasternodes;
    view.ForEachMasternode([&](const uint256 &mnId, CMasternode node) {
        if (node.IsActive(targetHeight, view) && node.mintedBlocks) {
            activeMasternodes.insert(mnId);
        }
        return true;
    });
```
to determine whether the masternode was valid when a proposal is over.

Reply:
This is addressed in https://github.com/DeFiCh/ain/pull/1725